tests/conftest.py

```diff
@@ -17,9 +17,21 @@
 
 import cript
 
+
+def _get_cript_tests_env() -> bool:
+    """
+    A little helper function, that ensure to mostly default to True.
+    Except if an explicit spelling of False is detected.
+    """
+    env_var = os.getenv("CRIPT_TESTS")
+    if env_var in ("false", "FALSE", "False"):
+        return False
+    return True
```

Comment:
```python
def _get_cript_tests_env() -> bool:
    """
    A little helper function, that ensure to mostly default to True.
    Except if an explicit spelling of False is detected.
    """
    # get env var string, convert to title case, compare that it is equal to the string "True"
    return os.getenv("CRIPT_TESTS").title().strip() == "True"
```

the current version would return True if `CRIPT_TESTS` has `FalSE`

Reply:
Correct, I don't think that is very likely though, that people spell it that way.

Reply:
I have a hard time accepting this, I made a PR for this issue, let me know what you think

https://github.com/C-Accel-CRIPT/Python-SDK/pull/253